cmds/build.js

```diff
@@ -13,11 +13,19 @@ module.exports = {
       alias: 'n',
       describe: 'Build for node usage',
       default: false
+    },
+    'enable-experimental-browser-builds': {
+      alias: 'eebb',
+      describe: 'Use experimental webpack config',
+      default: false
+    },
+    env: {
```

Comment:
Do we really need this? What happens if you specify both this + NODE_ENV in the environment itself?

Reply:
this is because babelrc needs a NODE_ENV set, so if the user doesnt set it this is the default.

it gets merged